src/components/HomeNavBar.tsx

```diff
@@ -0,0 +1,116 @@
+import ListItemIcon from '@material-ui/core/ListItemIcon';
+import ListItemText from '@material-ui/core/ListItemText';
+import PersonIcon from '@material-ui/icons/Person';
+import BuildIcon from '@material-ui/icons/Build';
+import ShoppingCartIcon from '@material-ui/icons/ShoppingCart';
+import ReportProblemIcon from '@material-ui/icons/ReportProblem';
+import LocationOnIcon from '@material-ui/icons/LocationOn';
+import WbSunnyIcon from '@material-ui/icons/WbSunny';
+import { Root, Bar, MenuButton, AccountButton, ListStyle, ToolBar, Typograph } from '../styles/TopNavStyles';
+
+type Anchor = 'left';
+
+export default function HomeNavBar() {
+    const [state, setState] = React.useState({
+        left: false,
+    });
+
+    const tabs = [{
+        route: "/customers",
+        icon: PersonIcon,
+        label: "Customers"
+      },{
+        route: "/inventory",
+        icon: ShoppingCartIcon,
+        label: "Inventory"
+      },{
+          route: "/maintenance",
+          icon: BuildIcon,
+          label: "Maintenance"
+      },{
+        route: "/incidents",
+        icon: ReportProblemIcon,
+        label: "Incidents"
+      }]
+
+    const tabs2 = [{
+        route: "/sites",
+        icon: LocationOnIcon,
+        label: "Sites"
+      },{
+        route: "/staff",
+        icon: WbSunnyIcon,
+        label: "Staff"
+      }]
+
+    const toggleDrawer = (anchor: Anchor, open: boolean) => (
+        event: React.KeyboardEvent | React.MouseEvent,
+      ) => {
+        if (
+          event.type === 'keydown' &&
+          ((event as React.KeyboardEvent).key === 'Tab' ||
+            (event as React.KeyboardEvent).key === 'Shift')
+        ) {
+          return;
+        }
+        setState({ ...state, [anchor]: open });
+    };
+
+    const list = (anchor: Anchor) => (
+        <ListStyle
+          onClick={toggleDrawer(anchor, false)}
+          onKeyDown={toggleDrawer(anchor, false)}
+        >
+          <List>
+        {tabs.map((tab, index) => (
+          <ListItem button key={tab.label}>
+            <ListItemIcon>{<tab.icon/>}</ListItemIcon>
+            <ListItemText primary={tab.label} />
+          </ListItem>
+        ))}
+      </List>
+      <Divider />
+      <List>
+        {tabs2.map((tab, index) => (
+          <ListItem button key={tab.label}>
+            <ListItemIcon>{<tab.icon/>}</ListItemIcon>
+            <ListItemText primary={tab.label} />
+          </ListItem>
+        ))}
+      </List>
+        </ListStyle>
+      );
+
+  return (
+    <Root>
+      <Bar>
+        <ToolBar>
+          <MenuButton>
+            <React.Fragment key={'left'}>
+                <MenuIcon onClick={toggleDrawer('left', true)}>{'left'}</MenuIcon>
+                <Drawer anchor={'left'} open={state['left']} onClose={toggleDrawer('left', false)}>
+                    {list('left')}
+                </Drawer>
+            </React.Fragment>
+          </MenuButton>
+          <Typograph variant="h4" noWrap>
```

Comment:
Since we want to universalize the top Nav Bar to exist in screens outside of the homepage, you can remove these Typography components! (Only include the Menu/Account buttons)